### PR TITLE
fix usd value on address page

### PR DESCRIPTION
**Pull request by ayrat555**
## Changelog

- fix usd value on address page

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -31,44 +31,44 @@ jobs:
 
       - restore_cache:
           keys:
-             - v7-mix-compile-{{ checksum "OTP_VERSION.lock" }}-{{ checksum "ELIXIR_VERSION.lock" }}-{{ checksum "mix.lock" }}
```
Comment (zachdaniel) [inline .circleci/config.yml]: Why these changes?

Comment (ayrat555) [reply .circleci/config.yml]: I thought tests failed because of stale cache
